### PR TITLE
MNT: Missing kwargs from axes.step.

### LINKED ISSUE #650: Missing kwargs in plot_stacked_bar_graph
### Description

I just saw this in the code and wanted to report it.  I think we need to pass in kawrgs to the below line in plot_stacked_bar_graph.

'''
my_hist, bins = np.histogram(xdata.values.flatten(), bins=bins, density=density)
'''

### PR DESCRIPTION
**Pull request by zssherman**
- [x] Closes #650 
- [x] PEP8 Standards or use of linter
- [x] Xarray Dataset or DataArray variable naming follows 'ds' or 'da' naming

### CODE REVIEW

**Comment by zssherman** [comment]
@mgrover1 Its adding a missing dictionary for histogram for range and weight parameters, think an example is unnecessary with the docs reflecting that. I do need to update the tests still though.

**Comment by zssherman** [comment]
Or do you mean like an example here?

**Comment by mgrover1** [comment]
> Or do you mean like an example here?

I think adding an example of a histogram in the example gallery using this functionality would be helpful

**Comment by zssherman** [comment]
@mgrover1 Ah gotcha, yeah I can do that.

**Review by mgrover1 — APPROVED** [review]
Great! Looks good! Thanks for adding that example @zssherman

File: examples/plotting/plot_hist_kwargs.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Plot a histogram of Met data.
+----------------------------------------------------
+
+This is a simple example for how to plot a histogram
+of Meteorological data, while using hist_kwargs parameter.
+
+Author: Zachary Sherman
+"""
+
+from matplotlib import pyplot as plt
```
Comment (mgrover1) [inline examples/plotting/plot_hist_kwargs.py:11]: You need more imports here - 
```python
import numpy as np
from act.plotting import HistogramDisplay
```

**Comment by mgrover1** [comment]
Once the tests pass, I think this is good to merge! Great work!

**Pull request merged**